src/server/status/metrics.js

```diff
@@ -1,27 +1,35 @@
 import _ from 'lodash';
 import Samples from './samples';
-module.exports = function (kbnServer, server, config) {
-  let lastReport = Date.now();
-
-  kbnServer.metrics = new Samples(12);
+import { keysToSnakeCaseShallow } from '../../utils/case_conversion';
 
+module.exports = function (kbnServer, server, config) {
   server.plugins['even-better'].monitor.on('ops', function (event) {
-    const now = Date.now();
-    const secSinceLast = (now - lastReport) / 1000;
-    lastReport = now;
-
     const port = config.get('server.port');
-    const requests = _.get(event, ['requests', port, 'total'], 0);
-    const requestsPerSecond = requests / secSinceLast;
-
-    kbnServer.metrics.add({
-      heapTotal: _.get(event, 'psmem.heapTotal'),
-      heapUsed: _.get(event, 'psmem.heapUsed'),
-      load: event.osload,
-      responseTimeAvg: _.get(event, ['responseTimes', port, 'avg']),
-      responseTimeMax: _.get(event, ['responseTimes', port, 'max']),
-      requestsPerSecond: requestsPerSecond
-    });
+    const timestamp = new Date().toISOString();
+    kbnServer.metrics = {
+      last_updated: timestamp,
+      collection_time_in_millis: config.get('ops.interval'),
```

Comment:
I really dislike shorthand, but it appears this is the standard set in ES. Thoughts on `collection_interval_in_millis`? I feel like we can make it a bit more descriptive.